ports/glutin/window.rs

```diff
@@ -260,13 +260,22 @@ impl Window {
             Event::Resized(width, height) => {
                 self.event_queue.borrow_mut().push(WindowEvent::Resize(Size2D::typed(width, height)));
             }
-            Event::MouseInput(element_state, mouse_button, _) => {
-                // FIXME(#11130, pcwalton): The third field should be used.
+            Event::MouseInput(element_state, mouse_button, pos) => {
                 if mouse_button == MouseButton::Left ||
-                                    mouse_button == MouseButton::Right {
-                        let mouse_pos = self.mouse_pos.get();
-                        self.handle_mouse(mouse_button, element_state, mouse_pos.x, mouse_pos.y);
-                   }
+                    mouse_button == MouseButton::Right {
```

Comment:
I think the problem is here, the `mouse_button` here should match up with the last line's `mouse_button` columns, and the `match` expression on the next line should be de-indented once.